Rules/CommonScripts/HelpfulDeathTips.cfg

```diff
@@ -23,25 +23,25 @@ tips = To hide these tips, and other in-game help, hit F1.;
 		Pay attention to your minimap for enemy tunnelers.;
 		Don't let siege weapons fall into the wrong hands!;
 		Immobilise siege weapons to prevent them from accidentally falling off a cliff!;
-		Press [SPACE] to light and throw a bomb, [C] to pickup.;
+		Press [SPACE] to light a bomb, [C] or [SPACE] to throw it.;
 		Keep your ballistae safe - if an enemy takes it, they can use it against you!;
 		Drills can be cooled down by jumping into water or running around.;
 		Enemies can use your workshops. Don't let them buy explosives in your base!;
 		Teamwork is the key to success!;
-		Try to help your team mates if the situation gets tight.;
+		Try to help your team mates.;
 		As a knight, you can put your shield up in the air to glide.;
 		Wood blocks are cheaper than stone blocks, but are also weaker and can catch fire.;
-		1 bomb can deal up to 3 hearts of damage.;
-		Drills are a fast way to dig, but only yield 50%. Don't use them on precious gold or during build time!;
+		A bomb can deal up to 3 hearts of damage.;
+		Drills only yield 75% of stone, but 100% of gold.;
```

Comment:
It might not be wise to include such specific information considering drills are constantly being tweaked.

Reply:
Before this tip said that drills give 50% stone and destroy gold. Even if mat yield for stone will be changed (which i doubt because most players seem fine with how drills currently work), we can always update the tip.